### PR TITLE
add data from aoe library

### PR DESCRIPTION
**Pull request by collinstommy**
- Update descriptions
- add data from https://github.com/collinstommy/aoe-library/blob/main/src/data.json

To Do:
- Add screen shots to `/previews`

### CODE REVIEW

File: data/community_pages.yaml [hunk 1]
```diff
@@ -324,3 +324,208 @@
...
+  preview: ""
+  contact:
+    - aoezone: ~
+    - email: ~
+    - twitter: ~
+    - discord: ~
+    - form: ~
+    - github: ~
+  repository: ~
+- url: "aoezone.net"
```
Comment (happyleavesaoc) [inline data/community_pages.yaml:398]: Already exists

Comment (collinstommy) [reply data/community_pages.yaml]: removed

**Comment by simonsan** [comment]
Great!

File: data/community_pages.yaml [hunk 2]
```diff
@@ -324,3 +324,194 @@
...
+  preview: ""
+  contact:
+    - aoezone: ~
+    - email: ~
+    - twitter: ~
+    - discord: ~
+    - form: ~
+    - github: ~
+  repository: ~
+- url: "aoe2.se"
```
Comment (happyleavesaoc) [inline data/community_pages.yaml:398]: Same as `siegeengineers.org`

File: data/community_pages.yaml [hunk 3]
```diff
@@ -324,3 +324,194 @@
...
+  preview: ""
+  contact:
+    - aoezone: ~
+    - email: ~
+    - twitter: ~
+    - discord: ~
+    - form: ~
+    - github: ~
+  repository: ~
+- url: "aoe2-de-tools.herokuapp.com/villagers-required.html"
```
Comment (happyleavesaoc) [inline data/community_pages.yaml:471]: Duplicated

File: data/community_pages.yaml (hunk 2 above)
Comment (simonsan) [reply data/community_pages.yaml:398]: I think it wouldn't make sense to have something as `aliases`, just one page of them is enough? Thoughts?

Comment (collinstommy) [reply data/community_pages.yaml]: Removed the duplicate

**Comment by happyleavesaoc** [comment]
Merged, thanks!

**Comment by simonsan** [comment]
@happyleavesaoc I think the preview pictures were still missing in this PR

**Comment by happyleavesaoc** [comment]
@simonsan oops, missed that. I think it's ok to update later.

**Pull request merged**